src/main/resources/resources/logisim/strings/soc/soc_ja.properties

```diff
@@ -457,16 +444,7 @@ SocVgaComponent = VGAスクリーン
 #
 VgaBufferAddress = ピクセルバッファのアドレス:
 VgaInitialDisplayMode = 初期モード:
-VgaMode1024x768 = 1024x768
-VgaMode160x120 = 160x120
-VgaMode320x240 = 320x240
-VgaMode640x480 = 640x480
-VgaMode800x600 = 800x600
-VgaSoft1024x768 = Software 1024x768:
-VgaSoft160x120 = Software 160x120:
-VgaSoft320x240 = Software 320x240:
-VgaSoft640x480 = Software 640x480:
-VgaSoft800x600 = Software 800x600:
+#VgaSoftMode = Software %s:
```

Comment:
Is it mend to be commented out?
If yes, please use `# ==>`.

Reply:
Yes. I got no clue how this should be in Japanese, so it's better to fallback to English for this one.